utils/dbconnector.ts

```diff
@@ -0,0 +1,31 @@
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { Pool } from "pg";
+import { IDecodableConfigPostgreSQL } from "./config";
+
+const pool = (
+  dbUser: NonEmptyString,
+  dbPassword: NonEmptyString,
+  dbHost: NonEmptyString,
+  dbPort: NonEmptyString,
+  dbDatabaseName: NonEmptyString,
+  dbIdleTimeout: number = 30000
+  // eslint-disable-next-line max-params
```

Comment:
Rule of thumb: when a function expects more than 3 parameters, maybe it's worth considering use a configurator object. Example:
```ts
// pool now has just one argument
const pool = (
  params: {
    dbUser: NonEmptyString;
    dbPassword: NonEmptyString;
    dbHost: NonEmptyString;
    dbPort: NonEmptyString;
    dbDatabaseName: NonEmptyString;
    dbIdleTimeout: number = 30000
  }
) => Pool
  
// alternative: cluster parameters that goes together
const pool = (
  db: { name: NonEmptyString; host: NonEmptyString; port: NonEmptyString },
  credentials: { user: NonEmptyString; password: NonEmptyString; },
  options: { idleTimeout: number; } = { idleTimeout = 30000 }
) => Pool  
```

Reply:
Yes,

very nice advice. It's more clear and readable.

Fix with 068aab1